commands/game.py

```diff
@@ -0,0 +1,935 @@
+                    song_length = song_name_api[x]['length']
+                    song_length = strftime("%H:%M:%S", gmtime(song_length))
+                    if fever:
+                        song_weight_list.append(
+                            [song_name + '(SP)', round(((song_values[2] + song_values[3] * 2) * 1.1) * 100),
+                             song_length])
+                    else:
+                        song_weight_list.append(
+                            [song_name + '(SP)', round(((song_values[0] + song_values[1] * 2) * 1.1) * 100),
+                             song_length])
+                song_values = song_meta_api[x]["3"]["7"]
+                try:
+                    if song_name_api[x]['musicTitle'][1] is not None:
+                        song_name = song_name_api[x]['musicTitle'][1]
+                    else:
+                        song_name = song_name_api[x]['musicTitle'][0]
+                except KeyError:
+                    song_name = song_name_api[x]['musicTitle'][0]
+                song_length = song_name_api[x]['length']
+                song_length = strftime("%H:%M:%S", gmtime(song_length))
+                if fever:
+                    song_weight_list.append(
+                        [song_name, round(((song_values[2] + song_values[3] * 2) * 1.1) * 100), song_length])
+                else:
+                    song_weight_list.append(
+                        [song_name, round(((song_values[0] + song_values[1] * 2) * 1.1) * 100), song_length])
+            if song_weight_list:
+                song_weight_list = sorted(song_weight_list, key=itemgetter(1), reverse=True)
+                song_weight_list = song_weight_list[:20]
+                if fever:
+                    title = "Song Meta (with Fever)"
+                else:
+                    title = "Song Meta (no Fever)"
+                output = ("```" + title + "\n\n" + tabulate(song_weight_list, tablefmt="plain",
+                                                            headers=["Song", "Score %", "Length"]) + "```")
+        return output
+
+    @game_commands.command(name='songmeta',
+                           description='Show song meta info')
+    async def song_meta(self,
+                        ctx: discord.ApplicationContext,
+                        fever: Option(bool, "Whether or not fever is enabled", required=False, default=True),
+                        songs: Option(str, "Song names to lookup. Separate multiple song names with a comma.",
+                                      required=False)):
+        await ctx.interaction.response.defer()
+        if songs:
+            songs = songs.replace(', ', ',')
+            song_list = songs.split(',')
+            song_meta = await self.get_song_meta_output(fever, song_list)
+            await ctx.interaction.followup.send(song_meta)
+        else:
+            song_meta = await self.get_song_meta_output(fever)
```

Comment:
Gonna keep this formatted with tabulate instead of embed since embeds don't guarantee a number of inline fields per row (on my phone, it's one field per row)